rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
#1267 should probably guide the options taken by this service so that it and metadata-service have a similar interface. 

When I think it over more, I think api-service should take a single option, 
`--listen=addr:port`, where `addr` is either an IPv4 address, or 0.0.0.0, representing all interfaces, and where `port` is a TCP port number. The default should be `--listen=0.0.0.0:15441`. This behavior existed in the previous revision, with a longer option name.

This would in turn suggest: 1) Changing the api-service.md doc to match the new flag. 2) Subsequent PR to metadata-service to change its current `listen-port` option to be consistent with this proposed `--listen`.

The wider talk of socket activation on systemd systems probably doesn't fit in this PR.

Sorry for my indecision, but I'm just sort of catching up to the history of this discussion in the metadata-service.